apps/studio/state/tabs.tsx

```diff
@@ -399,8 +400,16 @@ export type TabsState = ReturnType<typeof createTabsState>
 export const TabsStateContext = createContext<TabsState>(createTabsState(''))
 
 export const TabsStateContextProvider = ({ children }: PropsWithChildren) => {
-  const project = useSelectedProject()
-  const state = useConstant(() => createTabsState(project?.ref ?? ''))
+  const { data: project } = useSelectedProjectQuery()
+
+  const initState = useConstant(() => createTabsState(project?.ref ?? ''))
```

Comment:
You don't need the `useConstant`, you can just instantiate the tabsState in the `useState`, right?